### PR TITLE
Docs: separating css from plugin imports.

### LINKED ISSUE #16486: Inconsistent spacing between imports
## 📝 Provide a description of requested docs changes

The spacing between imports in docs snippets is not always consistent, for example:
- In https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/getting-started/installation/quick-start.html, there's no empty line above style imports:
empty-line-quick-start

- In integration guides - e.g. https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/getting-started/installation/angular.html, the line is present:
empty-line-angular

- In https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/getting-started/setup/ui-language.html, there's no empty line above translation imports:
ui-language-translations

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: separating css from plugin imports. Closes #16486

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/tutorials/widgets/data-from-external-source.md [hunk 1]
```diff
@@ -107,24 +107,25 @@ Finally, you need to load the `ExternalDataWidget` plugin in your `main.js` file
...
+		console.log( 'Editor was initialized', editor );
+
+		CKEditorInspector.attach( { editor: 'editor' } );
+
+		// Expose for playing in the console.
+		window.editor = editor;
+	} )
+	.catch( error => {
+		console.error( error.stack );
+	} );
```
Comment (gorzelinski) [inline docs/tutorials/widgets/data-from-external-source.md:128]: It's hard to tell what's going on here (and in other, similar snippets) - the change looks the same on the preview. Are those tabs changed to spaces or the other way around?

**Review by gorzelinski — APPROVED** [review]
Looks fine. I just added one question.

File: docs/tutorials/widgets/data-from-external-source.md (hunk 1 above)
Comment (godai78) [reply docs/tutorials/widgets/data-from-external-source.md:128]: Spaces to tabs. We use tabs in docs, globally.

Comment (gorzelinski) [reply docs/tutorials/widgets/data-from-external-source.md:128]: Ok, I think I see - it's deeper nesting.

**Pull request merged**